helm/app/values.yaml

```diff
@@ -1,3 +1,21 @@
+namespace: go-api-boilerplate
```

Comment:
i think we should use value that is passed with flag while doing
`helm install go-api-boilerplate helm/app/ --namespace go-api-boilerplate`

Reply:
```
namespace: {{ .Values.namespace }}
```

Reply:
Following best practices we can keep it globally as `{{ $.Release.Namespace }}` so that it's in line with helm install parameters. 
Reference: [release namespace value VS actual deployment namespace](https://github.com/helm/helm/issues/4878)